Source/WebCore/html/shadow/attachmentElementShadow.css

```
@@ -48,22 +52,30 @@ div#attachment-container {
 }
 
 div#attachment-preview-area {
-    width: calc(var(--icon-size) + var(--icon-margin));
     display: grid;
     align-items: center;
     justify-items: end;
     flex-shrink: 0;
+    margin: var(--icon-margin);
 }
 
 img#attachment-icon,
 div#attachment-placeholder,
 div#attachment-progress {
     grid-area: 1 / 1;
-    width: var(--icon-size);
-    aspect-ratio: 1;
+    align-self: center;
+    justify-self: center;
+}
+
+div#attachment-placeholder,
+div#attachment-progress {
+    width: calc(3/4 * var(--icon-size));
+    height: calc(3/4 * var(--icon-size));
 }
 
 img#attachment-icon {
+    max-width: var(--icon-size);
+    max-height: var(--icon-size);
```

```suggestion
    height: var(--icon-size);
```

I would ensure the height is actually filled.

---

but in the case of a wide and short icon, I wouldn't want the height to be filled, right?

---

we discussed this offline, and it seems like there is the possibility for wide icons, so this is fine